mod-di-converter-storage/src/main/resources/folijet/mod-di-converter-storage/features/jobProfiles.feature

```diff
@@ -698,4 +698,379 @@ Feature: Job Profiles
+                "fields": [
+                  {
+                    "label": "field",
+                    "value": "010"
+                  },
+                  {
+                    "label": "indicator1",
+                    "value": ""
+                  },
+                  {
+                    "label": "indicator2",
+                    "value": ""
+                  },
+                  {
+                    "label": "recordSubfield",
+                    "value": "z"
+                  }
+                ],
+                "staticValueDetails": null,
+                "dataValueType": "VALUE_FROM_RECORD"
+              },
+              "existingRecordType": "INSTANCE",
+              "existingMatchExpression": {
+                "fields" : [ {
+                  "label" : "field",
+                  "value" : "instance.identifiers[].value"
+                }, {
+                  "label" : "identifierTypeId",
+                  "value" : "c858e4f2-2b6b-4385-842b-60532ee34abb"
+                } ],
+                "dataValueType": "VALUE_FROM_RECORD"
+              },
+              "matchCriterion": "EXACTLY_MATCHES"
+            }
+          ],
+          "existingRecordType": "INSTANCE"
+        },
+        "addedRelations": [],
+        "deletedRelations": []
+      }
+      """
+    When method POST
+    Then status 201
+    * def instanceMatchProfileId = $.id
+
+    # Create job profile
+    * def jobProfileUpdateName = "FAT-13630_3: Job profile"
+    Given path 'data-import-profiles/jobProfiles'
+    And headers headersUser
+    And request
+      """
+      {
+        "profile": {
+          "name": "#(jobProfileUpdateName)",
+          "description": "",
+          "dataType": "MARC"
+        },
+        "addedRelations": [
+          {
+            "masterProfileId": null,
+            "masterProfileType": "JOB_PROFILE",
+            "detailProfileId": "#(instanceMatchProfileId)",
+            "detailProfileType": "MATCH_PROFILE",
+            "order": 0
+          },
+          {
+            "masterProfileId": "#(instanceMatchProfileId)",
+            "masterProfileType": "MATCH_PROFILE",
+            "detailProfileId": "#(instanceActionProfileId)",
+            "detailProfileType": "ACTION_PROFILE",
+            "order": 0,
+            "reactTo": "MATCH"
+          }
+        ],
+        "deletedRelations": []
+      }
+      """
+    When method POST
+    Then status 201
+    * def updateJobProfileId = $.id
+
+    # Update job profile
+    * print 'Update Job profile with remove action profile'
+    Given path 'data-import-profiles/jobProfiles', updateJobProfileId
+    And headers headersUser
+    And request
+    """
+    {
+      "profile": {
+        "name": "FAT-13630_3: Job profile update",
+        "description": "",
+        "dataType": "MARC"
+      },
+      "addedRelations": [],
+      "deletedRelations": [
+      {
+        "detailProfileId": "fa45f3ec-9b83-11eb-a8b3-0242ac130003",
```

Comment:
should we use here #(instanceActionProfileId) instead of hardcoded id to specify id of the created action profile, which is used to create the match-to-action association?

Reply:
yes, good catch, thanks Ruslan!

Reply:
got it, thanks!